projects/vdk-plugins/vdk-csv/src/vdk/plugin/csv/csv_plugin.py

```diff
@@ -83,6 +93,65 @@ def ingest_csv(ctx: Context, file: str, table_name: str, options: str) -> None:
     click.echo(f"Ingesting csv file {file} finished.")
 
 
+@hookimpl
+def vdk_configure(config_builder: ConfigurationBuilder) -> None:
+    """
+    Here we define what configuration settings are needed for sqlite with reasonable defaults
+    """
+    sqlite_configuration.add_definitions(config_builder)
+
+
+@hookimpl
+def initialize_job(context: JobContext) -> None:
+    conf = SQLiteConfiguration(context.core_context.configuration)
+    context.connections.add_open_connection_factory_method(
+        "SQLITE",
+        lambda: SQLiteConnection(pathlib.Path(conf.get_sqlite_file())).new_connection(),
+    )
+
+    context.ingester.add_ingester_factory_method(
+        "sqlite", (lambda: IngestToSQLite(conf))
+    )
```

Comment:
We'd like the export-csv command to work with any database that we have configured, not just SQLite; this entire chunk should be dropped I believe.